dumb-jump.el

```diff
@@ -699,7 +699,7 @@ or most optimal searcher."
            :tests ("newtype Test a = Something { b :: Kek }"
                    "data Test a b = Somecase a | Othercase b"
                    "type family Test (x :: *) (xs :: [*]) :: Nat where"
-                   "data family Test"
+                   "data family Test "
```

Comment:
Is this space on purpose?

Reply:
Yes. I believe that every identifier should be followed by a space or eof. `\b` doesn't work because in haskell we have a lot of things like `Identifier` and `Identifier'`, so matching first will match second also.